tklinenums/tklinenums.py

```diff
@@ -5,12 +5,12 @@
 from tkinter import Canvas, Event, Misc, Text
 from tkinter.font import Font
 
+
 system: str = str(platform.system())
-if system == "Darwin":
+if system == "Darwin" or "Windows":
```

Comment:
This is buggy, since it's equivalent to this: `if (system == "Darwin") or "Windows":` and since the string "Windows" is truthy, this condition will always be True.
The correct statement would be `if system in ("Darwin", "Windows"):`

Reply:
I can fix this on main.

Reply:
thanks